developer_manual/basics/storage/database.rst

```diff
@@ -322,18 +322,19 @@ Slugs are used to identify resources in the URL by a string rather than integer
     $author->slugify('name');  // Some-thing
 
 Table management tips
--------------------------
+---------------------
 
 It makes sense to apply some general tips from the beginning, so you don't have to migrate your data and schema later on.
 
-1. Don't use table name longer than 23 characters. As Oracle is limited to 30 chars and we need 3 more for `oc_` at the beginning and 5 for the primary key suffix `_pkey`.
+1. Don't use table name longer than 23 characters. As Oracle is limited to 30 chars and we need 3 more for ``oc_`` at the beginning and 5 for the primary key suffix ``_pkey``.
 
-2. Add an auto-incremented `id` column. This will ease the use of `QBMapper` + `Entity` approach:
+2. Add an auto-incremented ``id`` column. This will ease the use of ``QBMapper`` + ``Entity`` approach:
     - https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Db/QBMapper.php
     - https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Db/Entity.php
```

Comment:
Makes sense to have it render locally then you can see the syntax does something unintended:

Before | After
---|---
![Bildschirmfoto vom 2022-11-08 13-55-40](https://user-images.githubusercontent.com/213943/200570602-1b2e1dec-4534-4e92-9ac0-f35c613192ca.png) | ![grafik](https://user-images.githubusercontent.com/213943/200570577-668a672e-071b-4091-8da4-06e37e149ba8.png)

```suggestion
2. Add an auto-incremented ``id`` column. This will ease the use of ``QBMapper`` + ``Entity`` approach:

  - https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Db/QBMapper.php
  - https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Db/Entity.php
```

Reply:
Thanks, fixed :)